JSONPlaceholderTests/Tests.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using JSONPlaceholderAPIClient;
+using NUnit.Framework;
+
+namespace JSONPlaceholderTests
+{
+    [TestFixture]
+    public class Tests
+    {
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+           ApiClient.Put();
+        }
+
+        [Test]
+        public async Task CreatePost_ShouldReturnSuccessfulCode()
+        {
+            CreatePostModel post = new CreatePostModel(999, "new title", "same text");
+
+            var response = await ApiClient.CreatePosts(post);
+
+            Assert.AreEqual(response.StatusCode, HttpStatusCode.Created);
+        }
+
+        [Test]
+        public async Task CreatePost_ShouldReturnFullNewPost()
+        {
+            CreatePostModel post = new CreatePostModel(999, "new title", "same text");
+
+            var response = await ApiClient.CreatePosts(post);
+
+            Assert.IsNotEmpty(response.DeserealizedContent.Result.ToString());
+        }
+
+        [Test]
+        public async Task GetPost_ShouldHaveNotNullPost_UsingExistId()
+        {
+            var response = await ApiClient.GetPost("1");
+
+            Assert.IsNotNull(response.DeserealizedContent.Result.Body);
+        }
+
+        [Test]
+        public async Task GetPost_ShouldHaveSuccessfullCode_UsingExistId()
+        {
+            var response = await ApiClient.GetPost("10");
+
+            Assert.AreEqual(response.StatusCode, HttpStatusCode.OK);
+        }
+
+        [Test]
+        public async Task GetPost_ShouldHaveNoSuccessfulCode_UsingNoExistId()
+        {
+            var response = await ApiClient.GetPost("0");
+
+            Assert.AreEqual(response.StatusCode, HttpStatusCode.NotFound);
+        }
+
+        [Test]
+        public async Task GetPost_ShouldHaveNotEmptyBody_UsingExistId()
+        {
+            var response = await ApiClient.GetPost("4");
+
+            Assert.IsNotEmpty(response.DeserealizedContent.Result.Title);
+        }
+
+        [Test]
+        public async Task GetPost_ShouldHaveSomeTitle_UsingExistId()
+        {
+            string title = "qui est esse";
+            var response = await ApiClient.GetPost("2");
+
+            Assert.AreEqual(response.DeserealizedContent.Result.Title, title);
+        }
+
+        [Test]
+        public async Task GetAllPosts_ShouldBeNotNullPosts_PostsExist()
+        {
+            var response = await ApiClient.GetAllPosts();
```

Comment:
If you need to verify in to different test cases same response it makes sense to store response in TestFixture field that you will initialize only one in OneTimeSetUp